Utils/PluginActivator/source/COMRPCStarter.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2020 Metrological
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#include "Module.h"
+
+#include "IPluginStarter.h"
+
+using namespace WPEFramework;
+
+/**
+ * @brief COM-RPC implementation of a plugin starter
+ *
+ * Connects to Thunder over COM-RPC and attempts to start a given plugin
+ */
+class COMRPCStarter : public IPluginStarter {
+public:
+    explicit COMRPCStarter(const string& pluginName);
+    ~COMRPCStarter() override;
+
+    bool activatePlugin(const int maxRetries, const int retryDelayMs) final;
```

Comment:
Lets have some C++ discussions next week as well... Why is the final applicable on this method call ?

Reply:
No real need for `final` here tbh, switched to just using `override`